cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -146,6 +148,62 @@ type Member struct {
 	PeerURL string
 }
 
+// GetMemberID returns a member id of the given peer url
```

Comment:
`a member` -> `the member`
`id` -> `ID`
`url` -> `URL`

Reply:
Done.